com.alizadicomviewer.AlizaMS.svg

```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
There is an SVG icon upstream. 

https://github.com/AlizaMedicalImaging/AlizaMS/tree/master/package/archive/usr/share/icons/hicolor/scalable/apps

Why adding this?